### DISCUSSION TITLE
[SUGGESTION] Inertia option

### DISCUSSION THREAD

**Discussion by computer-logix** [TiagoSilvaPereira/vemto-issues#141] [Ideas & Suggestions]
I program exclusively with Laravel using Inertia, so other than making the schema and dumping it into my own project I haven't used your product yet for any actual work.     Your Kanban shows an entry for inertia support as a 'maybe'   I certainly hope that changes someday to a feature request!   

If anyone else uses Laravel 8 Inertia mention it here so it gets on the development timeline.

**Comment by deanlj** [comment]
I would add that now jetstream is being used much more, the Intertia and LiveWire options would be in quite high demand.

I personally use LiveWire more than inertia but it would make sense for Vemto to generate interia or livewire depending on Jetstream version

**Comment by computer-logix** [comment]
So I take it Inertia is a dead topic considering Tiago closed it a year and a half ago?
Too bad...   I purchased a lifetime license and haven't even used it yet.   All my clients want SPA apps and inertia works so well.

**Comment by TiagoSilvaPereira** [comment]
@computer-logix, it is still in my plans. The problem is that I'm stuck with the Vemto 1.x architecture, so I'm creating Vemto 2.x with better architecture to support multiple stacks (the idea behind Vemto 2.x is to be less invasive and more reactive, so you can use whatever stack you want... Vemto 2.x is under active development, and I'm sharing news almost weekly on my personal Twitter profile: https://twitter.com/Tiago_Ferat)

**Reply by computer-logix** [reply]
Awesome!   Thanks for the update.   So far, I've only been using Vemto for quick migration file creation.   I'll be patient!
